std-bits/base/src/main/java/org/enso/base/enso_cloud/audit/AuditLog.java

```diff
@@ -13,16 +13,27 @@
  * the meantime, all waiting messages (up to some limit) will be sent in a single request.
  */
 public final class AuditLog {
+  private AuditLog() {}
+
+  private static AuditLogApiAccess apiAccess;
+
+  private static AuditLogApiAccess apiAccess() {
```

Comment:
Why lazy initialization? Make the field `private static final` and initialize it in class initializer method.

Reply:
@Akirathan you wrote somewhere

> Generalizing LogApiAccess as done

what exactly is generalized here? Instead of using `AuditLogApiAccess.INSTANCE` you call `apiAccess()`. Where is the generalization?

Reply:
I am just adding new `TelemetryApiAccess`, inspired by [AuditLogApiAccess](https://github.com/enso-org/enso/blob/58bbd85e488f94e2e7a7eedacfb9a544ec163278/std-bits/base/src/main/java/org/enso/base/enso_cloud/audit/AuditLogApiAccess.java). I have not yet added it to this PR.